Microsoft.Toolkit.Uwp.UI.Controls.Lottie/LottieDrawable.cs

```diff
@@ -0,0 +1,1090 @@
+// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED �AS IS�, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using Microsoft.Graphics.Canvas;
+using Microsoft.Graphics.Canvas.UI.Xaml;
+using Microsoft.Toolkit.Uwp.UI.Controls.Lottie.Animation.Content;
+using Microsoft.Toolkit.Uwp.UI.Controls.Lottie.Manager;
+using Microsoft.Toolkit.Uwp.UI.Controls.Lottie.Model;
+using Microsoft.Toolkit.Uwp.UI.Controls.Lottie.Model.Layer;
+using Microsoft.Toolkit.Uwp.UI.Controls.Lottie.Parser;
+using Microsoft.Toolkit.Uwp.UI.Controls.Lottie.Utils;
+using Microsoft.Toolkit.Uwp.UI.Controls.Lottie.Value;
+using Windows.Foundation;
+using Windows.UI;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls.Lottie
+{
+    /// <summary>
+    /// This can be used to show an lottie animation in any place that would normally take a drawable.
+    /// If there are masks or mattes, then you MUST call <seealso cref="RecycleBitmaps()"/> when you are done
+    /// or else you will leak bitmaps.
+    /// <para>
+    /// It is preferable to use <seealso cref="LottieAnimationView"/> when possible because it
+    /// handles bitmap recycling and asynchronous loading
+    /// of compositions.
+    /// </para>
+    /// </summary>
+    public class LottieDrawable : UserControl, IAnimatable, IDisposable
```

Comment:
@azchohfi Was this the piece we needed to make it into a composition brush?  If so, should part of these be in a namespace outside of `controls` so it makes more sense when referenced in `Microsoft.Toolkit.UWP.UI.Media`?

Reply:
Which namespace you have in mind? I've refactored the method so it is static and It don't have any dependencies on LottieDrawable itself, only BitmapCanvas, CompositionLayer and LottieComposition (so where ever it goes, it will have a dependency on Lottie classes). I can put it anywhere we want, but we will have to move a lot of classes to make this independent of **controls**.

Reply:
Actually, thinking about it more, a brush would just ship in the same package anyway for convenience, so don't think we have to worry about it for now.  If we need to change anything, we can figure it out when I start working on those brushes.